dipy/viz/actor.py

```diff
@@ -119,31 +117,31 @@ def slicer(data, affine=None, value_range=None, opacity=1.,
     set_input(image_resliced, im)
     image_resliced.SetResliceTransform(transform)
     image_resliced.AutoCropOutputOn()
-
     # Adding this will allow to support anisotropic voxels
     # and also gives the opportunity to slice per voxel coordinates
-
     RZS = affine[:3, :3]
     zooms = np.sqrt(np.sum(RZS * RZS, axis=0))
     image_resliced.SetOutputSpacing(*zooms)
-
     image_resliced.SetInterpolationModeToLinear()
     image_resliced.Update()
 
+    # Set the lookup table
     if nb_components == 1:
         if lookup_colormap is None:
             # Create a black/white lookup table.
-            lut = colormap_lookup_table((0, 255), (0, 0), (0, 0), (0, 1))
+            lut = colormap_lookup_table((value_range[0], value_range[1]),
```

Comment:
Ah here you change the range to be anything. Okay this again is a different approach. So, I wonder when you said a bug did really something failed or by reading the docstring you didn't see the purpose of the function? We could add to parameters one for data_range (what I was calling value_range) and one for interp_range what here you use as input to the colormap.